rpc/codec/server.go

```diff
@@ -24,11 +24,17 @@ package codec
 import (
 	"bufio"
 	"bytes"
+	"crypto/tls"
 	"fmt"
 	"io"
 	"net/rpc"
 
+	"github.com/cockroachdb/cockroach/base"
+	cockroach_proto "github.com/cockroachdb/cockroach/proto"
```

Comment:
that would be much better, but yes, this is rather outside the scope of this PR.